exercises/clock/example.js

```diff
@@ -1,11 +1,25 @@
 exports.at = at;
 
+var HOURS_IN_A_DAY = 24;
+var MINUTES_IN_AN_HOUR = 60;
+var MINUTES_IN_A_DAY = HOURS_IN_A_DAY * MINUTES_IN_AN_HOUR;
+var MILLIS_IN_A_MINUTE = 60 * 1000;
+var MILLIS_IN_AN_HOUR = MINUTES_IN_AN_HOUR * MILLIS_IN_A_MINUTE;
+var MILLIS_IN_A_DAY = HOURS_IN_A_DAY * MILLIS_IN_AN_HOUR;
+
+function makePositive(time, maxValue) {
+  time %= maxValue;
+  time += maxValue;
+  return time;
+}
+
 function at(hours, minutes) {
-  var min = 1000 * 60;
-  var hr = min * 60;
+  hours = makePositive(hours, HOURS_IN_A_DAY);
+  minutes = makePositive(minutes, MINUTES_IN_A_DAY);
 
   var clock = {};
-  var value = (~~hours * hr) + (~~minutes * min);
+  var value = (~~hours * MILLIS_IN_AN_HOUR) + (~~minutes * MILLIS_IN_A_MINUTE);
```

Comment:
Nice catch @matthewmorgan!!

They were not neccessary, but I needed to include

    minutes = minutes || 0;

at line 17 because the `minutes` parameter was optional, so `~~` helped to convert it to a number. 

Now, with your suggestion is much cleaner, thanks!!